plugins/woocommerce-admin/client/homescreen/mobile-app-modal/components/MobileAppLoginInfo.tsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import { QRCodeSVG } from 'qrcode.react';
+import React from '@wordpress/element';
+import interpolateComponents from '@automattic/interpolate-components';
+import { __ } from '@wordpress/i18n';
+import { recordEvent } from '@woocommerce/tracks';
+import { Link } from '@woocommerce/components';
+
+export const MobileAppLoginInfo = ( {
+	loginUrl,
+}: {
+	loginUrl: string | undefined;
+} ) => {
+	// TODO: update the minimum app version
+	return (
+		<div>
+			{ loginUrl && (
+				<div>
+					<QRCodeSVG value={ loginUrl } size={ 140 } />
+					<p>
+						{ __(
+							'The app version needs to be 15.7 or above to sign in with this link.',
```

Comment:
I realised that I can't test this flow because the latest version I can install is 15.6

Reply:
I went on with it for a bit then realised that I need to install app center device profile on my personal phone so I noped outta there 😅 I'll take it that it works!